comps/imagegen/triton/Dockerfile.base

```diff
@@ -0,0 +1,114 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu:jammy
+ARG ARTIFACTORY_URL=vault.habana.ai
+ARG VERSION=1.15.1
+ARG REVISION=15
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV GC_KERNEL_PATH=/usr/lib/habanalabs/libtpc_kernels.so
+ENV HABANA_LOGS=/var/log/habana_logs/
+ENV OS_NUMBER=2204
+ENV HABANA_SCAL_BIN_PATH=/opt/habanalabs/engines_fw
+ENV HABANA_PLUGINS_LIB_PATH=/opt/habanalabs/habana_plugins
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    apt-utils \
+    bc \
+    build-essential \
+    ca-certificates \
+    dkms \
+    ethtool \
+    gcc \
+    git \
+    gnupg \
+    gpg-agent \
+    graphviz \
+    libgl1 \
+    libgoogle-glog0v5 \
+    libjemalloc2 \
+    libpq-dev \
+    locales \
+    lsof \
+    make \
+    openssh-client \
+    openssh-server \
+    protobuf-compiler \
+    python3 \
+    python3-dev \
+    python3-pip \
+    python3-tk \
+    python3-venv \
+    unzip \
+    vim \
+    libkrb5-3 \
+    libgnutls30 \
+    wget && \
```

Comment:
I'd make sure every single one of these packages are absolutely required before adding them to the container.
Things like `vim` for example are usually not encouraged for container users unless we want people to use these containers as their development environment.

Reply:
Yeah - I think your other comment is valid.

I will look and see if I can replace the Habana base image with something managed by a service/owner outside of this repo. 

Our OHIO team should probably provide us a "habana-base" image for building model servers on top of.

Reply:
So I am going to ignore your comments on the inner triton server dockerfiles for now, and look into outsourcing this code and building on that instead.